src/app.js

```diff
@@ -38,61 +68,47 @@ export const start = async args => {
   }
 
   try {
-    const response = await getRepos(username);
-
-    const repositories = response.user.repositories.edges;
+    var userGraph = await getRepos(username);
+  } catch (err) {
+    // This error message from the GitHub API is especially ugly.
+    process.stderr.write(`Error: could not get user ${username}.\n`);
+    process.exit(1);
+  }
 
-    if (!repositories.length) {
-      throw new Error('No repositories found.');
+  try {
+    // Check user & repo
+    const user = userGraph.user;
+    if (!user) {
+      throw new Error('user not found.');
```

Comment:
Sample response from the github api explorer

```
{
  "data": {
    "user": null
  },
  "errors": [
    {
      "type": "NOT_FOUND",
      "path": [
        "user"
      ],
      "locations": [
        {
          "line": 2,
          "column": 9
        }
      ],
      "message": "Could not resolve to a User with the login of 'alskhe2'."
    }
  ]
}
```
Can we just grab the errors `message` and `type` as part of what we report back?